code/_onclick/hud/screen_objects.dm

```diff
@@ -356,7 +356,8 @@
 					usr << up_image
 					QDEL_IN(up_image, 12)
 				return
-			var/turf/T = GetAbove(usr)
+			var/turf/T1 = get_turf(usr)
+			var/turf/T = GET_TURF_ABOVE(T1)
```

Comment:
Could be simplified to GET_TURF_ABOVE(get_turf(usr))

Reply:
Actually, no it cannot, the macro is incompatible with having procs passed in, and would fail compilation:

`code\game\turfs\simulated\wall_attacks.dm:77:error: .: missing comma ',' or right-paren ')'`

This is why I used support vars to write it

"But can you not change the macro?" -> Possibly, but it's ported 1:1 for a reason: I want to keep it as aligned as possible so we don't have to throw effort to keep things updated and fix everything else that might not work with the changed one down the line, if that means I have to use some support vars, so be it, it's a fine trade for me